src/decaylanguage/data/decfile.lark

```diff
@@ -63,9 +72,9 @@ _WS: WS_INLINE
 MODEL_NAME_AND_WS.2: MODEL_NAME _WS+
 MODEL_NAME_AND_SC.2: MODEL_NAME _WS* _SEMICOLON+
 
-MODEL_NAME : "BaryonPCR"|"BCL"|"BGL"|"BT02PI_CP_ISO"|"BTO3PI_CP"|"BTOSLLALI"|"BTOSLLBALL"|"BTOXSGAMMA"|"BTOXSLL"|"CB3PI-MPP"|"CB3PI-P00"|"D_DALITZ"|"ETAPRIME_DALITZ"|"ETA_DALITZ"|"ETA_FULLDALITZ"|"ETA_PI0DALITZ"|"FLATQ2"|"GENERIC_DALITZ"|"GOITY_ROBERTS"|"HELAMP"|"HQET3"|"HQET2"|"HQET"|"ISGW2"|"ISGW"|"LbAmpGen"|"LLSW"|"MELIKHOV"|"OMEGA_DALITZ"|"PARTWAVE"|"PHSP"|"PI0_DALITZ"|"PROPSLPOLE"|"PVV_CPLH"|"PYCONT"|"PYTHIA"|"SLBKPOLE"|"SLN"|"SLPOLE"|"SSD_CP"|"SSD_DirectCP"|"SSS_CP"|"SSS_CP_PNG"|"SSS_CPT"|"STS_CP"|"STS"|"SVP_CP"|"SVP_HELAMP"|"SVP"|"SVS_CP_ISO"|"SVS_CPLH"|"SVS_CP"|"SVS_NONCPEIGEN"|"SVS"|"SVV_CPLH"|"SVV_CP"|"SVV_HELAMP"|"SVV_NONCPEIGEN"|"SVVHELCPMIX"|"TAUHADNU"|"TAULNUNU"|"TAUSCALARNU"|"TAUVECTORNU"|"TSS"|"TVP"|"TVS_PWAVE"|"VLL"|"VSP_PWAVE"|"VSS_BMIX"|"VSS_MIX"|"VSS"|"VUB"|"VVPIPI"|"VVP"|"VVS_PWAVE"|"YMSTOYNSPIPICLEO"|"YMSTOYNSPIPICLEOBOOST"
+MODEL_NAME : "BCL"|"BC_SMN"|"BC_TMN"|"BC_VHAD"|"BC_VMN"|"BGL"|"BLLNUL"|"BNOCB0TO4PICP"|"BNOCBPTO3HPI0"|"BNOCBPTOKSHHH"|"BQTOLLLL"|"BQTOLLLLHYPERCP"|"BSTOGLLISRFSR"|"BSTOGLLMNT"|"BS_MUMUKK"|"BT02PI_CP_ISO"|"BTO3PI_CP"|"BTODDALITZCPK"|"BTOSLLALI"|"BTOSLLBALL"|"BTOSLLMS"|"BTOSLLMSEXT"|"BTOVLNUBALL"|"BTOXELNU"|"BTOXSGAMMA"|"BTOXSLL"|"BToDiBaryonlnupQCD"|"BaryonPCR"|"CB3PI-MPP"|"CB3PI-P00"|"D0GAMMADALITZ"|"DToKpienu"|"D_DALITZ"|"D_hhhh"|"ETAPRIME_DALITZ"|"ETA_DALITZ"|"ETA_FULLDALITZ"|"ETA_LLPIPI"|"ETA_PI0DALITZ"|"FLATQ2"|"FLATSQDALITZ"|"FOURBODYPHSP"|"GENERIC_DALITZ"|"GOITY_ROBERTS"|"HELAMP"|"HQET"|"HQET2"|"HQET3"|"ISGW"|"ISGW2"|"KS_PI0MUMU"|"LLSW"|"LNUGAMMA"|"LQCD"|"Lb2Baryonlnu"|"Lb2plnuLCSR"|"Lb2plnuLQCD"|"LbAmpGen"|"MELIKHOV"|"OMEGA_DALITZ"|"PARTWAVE"|"PHI_DALITZ"|"PHSP"|"PHSPDECAYTIMECUT"|"PHSPFLATLIFETIME"|"PI0_DALITZ"|"PROPSLPOLE"|"PTO3P"|"PVV_CPLH"|"PYCONT"|"PYTHIA"|"SLBKPOLE"|"SLL"|"SLN"|"SLPOLE"|"SSD_CP"|"SSD_DirectCP"|"SSS_CP"|"SSS_CPT"|"SSS_CP_PNG"|"STS"|"STS_CP"|"SVP"|"SVP_CP"|"SVP_HELAMP"|"SVS"|"SVS_CP"|"SVS_CPLH"|"SVS_CP_ISO"|"SVS_NONCPEIGEN"|"SVVHELCPMIX"|"SVV_CP"|"SVV_CPLH"|"SVV_HELAMP"|"SVV_NONCPEIGEN"|"TAUHADNU"|"TAULNUNU"|"TAUOLA"|"TAUSCALARNU"|"TAUVECTORNU"|"THREEBODYPHSP"|"TSS"|"TVP"|"TVS_PWAVE"|"VLL"|"VSP_PWAVE"|"VSS"|"VSS_BMIX"|"VSS_MIX"|"VTOSLL"|"VUB"|"VVP"|"VVPIPI"|"VVS_PWAVE"|"XLL"|"YMSTOYNSPIPICLEO"|"YMSTOYNSPIPICLEOBOOST"|"doKm"|"imqp"
```

Comment:
I'm happy to work on some more tests if you point me to LHCb decfiles that contain the new keywords. From last time I checked I had not seen so many unknown to DecayLanguage :).

Reply:
Instead of hunting through this big string I called;

```
python -c "from decaylanguage.dec.enums import known_decay_models;print('|'.join([f'\"{m}\"' for m in sorted(known_decay_models)]))"
```

 so the order is whatever `sorted` decided. I can restore it to the order it exists as in `known_decay_models`

Reply:
I think it is safer and I get back to the thing separately. Thanks.

Reply:
Oh, reordered already 👍.